### PR TITLE
Pcc13

### PR DESCRIPTION
**Pull request by abkosar**
ATTENTION: before clicking "Create Pull Request" please submit some meta data, thanks!

Difficulty level (1-10): [4]
Estimated time spent (hours): [1hr30min]
Completed (yes/no): [yes]
I stretched my coding skills (if yes what did you learn?): [I learned how to utilize collections module without using pandas]
Other feedback (what can we improve?): []

### CODE REVIEW

File: 13/abkosar/collections_on_movie_data.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+def main():
+    '''This is a template, feel free to structure your code differently.
+    We wrote some tests based on our solution: test_directors.py'''
+    movies_by_directors = get_movies_by_director()
+    director_average_scores = get_average_scores(movies_by_directors.keys())
+    directors_sorted = sorted(director_average_scores.items(), key=lambda x: x[1], reverse=True)
+    print_results(directors_sorted)
+    
+if __name__ == "__main__":
+    main()
```
Comment (feoh) [inline 13/abkosar/collections_on_movie_data.py]: Again nit - pep8 demands that you have a newline at end of file. Most good Python editors will allow you to turn on PEP8 compliance as a visual error. Pycharm does a fabulous job at this, and so do emacs and vim with the right magic in place :)

Comment (bbelderbos) [reply 13/abkosar/collections_on_movie_data.py]: A blank line or a newline? Former would throws a `W391 blank line at end of file`, I don't see a pep violation above?

Comment (abkosar) [reply 13/abkosar/collections_on_movie_data.py]: Hey @feoh -- Yeah I usually use PyCharm but for this I used Jupyterlab. I did a new commit with newline EOF. Thanks

Comment (feoh) [reply 13/abkosar/collections_on_movie_data.py]: @bbelderbos You're right. @abkosar apologies.

**Pull request merged**